src/libsync/creds/httpcredentials.cpp

```diff
@@ -38,14 +38,13 @@ namespace OCC {
 Q_LOGGING_CATEGORY(lcHttpCredentials, "nextcloud.sync.credentials.http", QtInfoMsg)
 
 namespace {
-    const char userC[] = "user";
-    const char isOAuthC[] = "oauth";
-    const char clientCertBundleC[] = "clientCertPkcs12";
-    const char clientCertPasswordC[] = "_clientCertPassword";
-    const char clientCertificatePEMC[] = "_clientCertificatePEM";
-    const char clientKeyPEMC[] = "_clientKeyPEM";
-    const char authenticationFailedC[] = "owncloud-authentication-failed";
-    const char needRetryC[] = "owncloud-need-retry";
+const char userC[] = "user";
+const char clientCertBundleC[] = "clientCertPkcs12";
+const char clientCertPasswordC[] = "_clientCertPassword";
+const char clientCertificatePEMC[] = "_clientCertificatePEM";
+const char clientKeyPEMC[] = "_clientKeyPEM";
+const char authenticationFailedC[] = "owncloud-authentication-failed";
+const char needRetryC[] = "owncloud-need-retry";
```

Comment:
please keep indentation changes in separate commits

Reply:
auto clang-format again, will undo

Reply:
done